### ISSUE TITLE
[Enhancement] Dev Portal Gift Codes Additions and QoL Improvements! 

### ISSUE THREAD

**Issue by Jump-Suit** [discordapp/gamesdk-and-dispatch#10]
Transferred from Dev Feedback: 
https://support-dev.discordapp.com/hc/en-us/community/posts/360047899951--Enhancement-Dev-Portal-Gift-Codes-Additions-and-QoL-Improvements-

**JumpSuit**

As something that I noticed regarding Gift Codes on the Discord Dev Dashboard, once they are expired they don't get removed from the Gift Code Tab? Why is this or is this intentional? I'd like a proper Indication of If they are expired or not so they can either:
A. Get deleted Automatically.
B. Users of a Team can Manually Delete them from the List.
C. They can get filtered out through a search or checkbox for 'expired' or 'not-expired'

I can't simply have a whole list of created codes with no proper way of managing them outside of downloading .csv files.... They should be managed from the Tab Directly!

**Comment by rickmartensnl** [comment]
Also, be able to hide them when they are all redeemed. Or delete them even when they are not redeemed(Like #33 to remove the gift codes for reasons like this).

**Comment by TheGoldenCactus11** [comment]
Yep, it's needed, we should also be able to see how many have been claimed, and maybe what ones, by marking the CSV. The ability to delete them is KEY. Metal's dungeon got 25k keys leaked, and it caused big problems, because Discord would not de-activate them, one reason this is for, is people trading them, then deleting them to scam.
